ivy/functional/frontends/torch/linalg.py

```diff
@@ -122,3 +122,8 @@ def inv_ex(input, *, check_errors=False, out=None):
 @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16", "float16")}, "torch")
 def eig(input, *, out=None):
     return ivy.eig(input, out=out)
+
+
+@to_ivy_arrays_and_back
+def matmul(input, other, *, out=None):
+    return torch_frontend.matmul(input, other, out=out)
```

Comment:
We use ivy functions to implement frontend functions.
Example here maybe `ivy.matmul`

Reply:
Sure I will get it done right away. Thanks for your time

Reply:
I wrote it as torch_frontend.matmul because it was given that any implementation that is already present should be mentioned as an alias. 
![frontend](https://user-images.githubusercontent.com/51944874/211972655-07fdfe97-42fa-4b6c-8eaf-f4ff05dd266b.png)

Reply:
Okay I get it now you have to add it to the tests of the original function and I'll get this merged🙂